lua/pluginList.lua

```diff
@@ -324,4 +324,38 @@ return packer.startup(function()
          require("mappings").fugitive()
       end,
    }
+
+   use {
+     "glepnir/lspsaga.nvim",
+     after = "nvim-lspconfig",
+     config = function ()
+       require("plugins.others").saga()
+     end,
+     setup = function ()
+       require("mappings").saga()
+     end
+   }
+
+  use {
+     "phaazon/hop.nvim",
+     event = "BufRead",
+     cmd = {"HopWord", "HopLine", "HopChar1", "HopChar2", "HopPattern"},
+     config = function() require("plugins.others").hop() end,
+     setup = function() require("mappings").hop() end
+  }
+
+  use {
```

Comment:
Hmmmm, 
I'm worried that this plugin may be a little small in the functionality provided, especially given `Treesitter/LSP` highlight TODOs already. 

Would you elaborate on what this plugin provides? (I've read through it's Readme, but maybe I'm missing something)